pkgs/top-level/all-packages.nix

```diff
@@ -15051,9 +15051,10 @@ in
 
   flyway = callPackage ../development/tools/flyway { };
 
-  inherit (callPackages ../development/libraries/fmt { }) fmt_7;
+  inherit (callPackages ../development/libraries/fmt { }) fmt_7 fmt_8;
 
   fmt = fmt_7;
+  fmt8 = fmt_8;
```

Comment:
```suggestion
```

Reply:
Reading the code now, not quite sure why I added that in the first place. Anyhow, I've commited your suggestion :+1: